services/core/functions/review-lambda/index.ts

```diff
@@ -12,6 +14,29 @@ interface ReviewLambdasBody {
 logger.settings.minLevel = 4;
 
 export const main = async (event: APIGatewayProxyEvent) => {
+  if (event.body === null) {
+    return Promise.resolve({
+      statusCode: 400,
+      body: "The request does not contain a body as expected.",
+    });
+  }
+
+  const header = event.headers[GITHUB_SIGNATURE_HEADER_KEY];
+  if (header === undefined) {
+    return Promise.resolve({
+      statusCode: 401,
+      body: "No authentication token found.",
+    });
+  }
+
+  const authenticated = await authenticate(header, event.body);
+  if (!authenticated) {
+    return Promise.resolve({
+      statusCode: 401,
+      body: "Unauthorized.",
+    });
+  }
```

Comment:
We should build out some generic error responses and codes but this is fine for now.